### PR TITLE
Bump Cognitive Services SDK to 7.0

### PR DESCRIPTION
**Pull request by yangyuan**
Swagger change

https://github.com/Azure/azure-rest-api-specs/pull/7944
https://github.com/Azure/azure-rest-api-specs/pull/7415

### CODE REVIEW

File: sdk/cognitiveservices/Microsoft.Azure.Management.CognitiveServices/src/Generated/CognitiveServicesManagementClientExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            /// <param name='operations'>
+            /// The operations group for this extension method.
+            /// </param>
+            /// <param name='subdomainName'>
+            /// The subdomain name to use.
+            /// </param>
+            /// <param name='type'>
+            /// The Type of the resource.
+            /// </param>
+            public static CheckDomainAvailabilityResult CheckDomainAvailability(this ICognitiveServicesManagementClient operations, string subdomainName, string type)
```
Comment (isra-fel) [inline sdk/cognitiveservices/Microsoft.Azure.Management.CognitiveServices/src/Generated/CognitiveServicesManagementClientExtensions.cs]: Please write tests to cover this new operation thanks

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @yangyuan , could you add a link to the swagger spec review PR in the description?
Thanks

File: sdk/cognitiveservices/Microsoft.Azure.Management.CognitiveServices/src/Generated/CognitiveServicesManagementClientExtensions.cs (hunk 1 above)
Comment (yangyuan) [reply sdk/cognitiveservices/Microsoft.Azure.Management.CognitiveServices/src/Generated/CognitiveServicesManagementClientExtensions.cs]: added

**Comment by yangyuan** [comment]
Hi @isra-fel, I added one missing test.

But the CI is failing, I checked it is not related to the change. Its actually cause by other SDKs.

**Review by isra-fel — APPROVED** [review]
Cool

**Comment by isra-fel** [comment]
The failed data-plane CIs were triggered by mistake, none of data-plane code was touched. So I'll just merge the PR.

**Pull request merged**